test/routes/author/index.spec.js

```diff
@@ -0,0 +1,71 @@
+import assert from 'power-assert';
+import request from 'supertest';
+import app from 'app.js';
+import AutherModel from 'models/Author';
+
+describe('/routes/author', ()=> {
+  let response;
+
+  before((done)=> {
+    const author = new AutherModel({
+      name: '鳥山明',
+      lastModified: new Date(),
+    });
+
+    author.save((saveError)=> {
+      if (saveError) {
+        return done(saveError);
+      }
+      request(app)
+        .get('/author/1')
+        .end((err, ret)=> {
+          if (err) {
+            return done(err);
+          }
+          response = ret;
+          done();
+        });
+    });
+  });
+
+  it('200を返す', ()=> {
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use esnext option).